ci-operator/step-registry/windows/e2e/operator/test/windows-e2e-operator-test-ref.yaml

```diff
@@ -6,6 +6,9 @@ ref:
     requests:
       cpu: 1000m
       memory: 400Mi
+  dependencies:
+    - env: OPERATOR_IMAGE
+      name: windows-machine-config-operator-test
```

Comment:
Shouldn't `OPERATOR_IMAGE`  include the registry? Following the pattern `registry.svc.ci.openshift.org/ci-op-<input-hash>/stable:${component}` ?

where `${component}` is the name of the WMCO image (`windows-machine-config-operator-test`)

Reply:
@jrvaldes I don't think that's needed, it was just the format of IMAGE_FORMAT env var pleas see https://docs.ci.openshift.org/docs/architecture/ci-operator/#referring-to-images-in-tests